### DISCUSSION TITLE
Make installable as a PWA

### DISCUSSION THREAD

**Discussion by cassidyjames** [plausible/analytics#181] [Anything Else]
It would be awesome if Plausible was installable as a progressive web app (PWA); this would make it easier to check analytics on Android, Windows, etc. by giving us a dedicated icon on the home screen.

**Comment by cassidyjames** [comment]
Ah, there is probably some overlap with #93 here as well; one of my complaints with the current setup is that there is no icon when adding to my home screen with both Chrome and Firefox Preview on Android. The title was also the entire title tag from HTML instead of a shorter one like “Plausible”.

For a PWA at least, you can provide a bunch of great info in a JSON manifest: https://developer.mozilla.org/en-US/docs/Web/Manifest

I'm not sure if iOS will also use any of that or not.

**Comment by Tragio** [comment]
iOS doesn't like manifest but it will use the icon 🚀

**Comment by gruns** [comment]
Hey guys! Picking up from an email with @metmarkosaric : if helpful, check out Pep (https://pep.dev/).

Pep turns websites into a Progressive Web Apps (PWA) in one line of code and makes them installable, work offline, always up-to-date, and neck-snapping fast (optimized JS+CSS, responsive images, etc). Plausible may fall within Pep's free tier, too, so it'd be completely free

Full disclosure: I built Pep. :upside_down_face:

**Comment by siimots** [comment]
Can you consider adding simple manifest.json?

It would make "Add to Homescreen" experience much better on iOS and Android.

I made [PR for Uptime Kuma](https://github.com/louislam/uptime-kuma/pull/411). So you can see how it makes web app better with small code changes.

I have no experience in Elixir, so I cannot make PR myself.

Just manifest.json is good start for basic PWA experience, no service worker needed.

**Reply by Reedyn** [reply]
This is indeed a very simple addition that would make it much better!

**Comment by jaydanurwin** [comment]
HI there plausible team, recent convert here! I don't have any Elixir experience but I'm willing to open a PR to get this done. It's really only a JSON file and some images for the icons. Let me know!

**Reply by Reedyn** [reply]
Why not just open the PR, then they can just merge whenever they see it 😀

**Comment by alistair3149** [comment]
If no one is picking this up, I can look into it and submit a PR on Monday 😃
